metrics/size.table/table.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2022 Block, Inc.
+
+package sizetable
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/cashapp/blip"
+	"github.com/cashapp/blip/sqlutil"
+)
+
+const (
+	DOMAIN = "size.table"
+
+	opt_total         = "total"
+	OPT_SCHEMA_FILTER = "schema"
+)
+
+// Table collects table sizes for domain size.table.
+type Table struct {
+	db *sql.DB
+	// --
+	query map[string]string
+	total map[string]bool
+}
+
+// Verify collector implements blip.Collector interface.
+var _ blip.Collector = &Table{}
+
+// NewTable makes a new Table collector,
+func NewTable(db *sql.DB) *Table {
+	return &Table{
+		db:    db,
+		query: map[string]string{},
+		total: map[string]bool{},
+	}
+}
+
+// Domain returns the Blip metric domain name (DOMAIN const).
+func (t *Table) Domain() string {
+	return DOMAIN
+}
+
+// Help returns the output for blip --print-domains.
+func (t *Table) Help() blip.CollectorHelp {
+	return blip.CollectorHelp{
+		Domain:      DOMAIN,
+		Description: "Table sizes",
+		Options: map[string]blip.CollectorHelpOption{
+			opt_total: {
+				Name:    opt_total,
+				Desc:    "Returns total size of all tables",
+				Default: "yes",
+				Values: map[string]string{
+					"yes": "Includes total size of all tables",
+					"no":  "Excludes total size of all tables",
+				},
+			},
+			OPT_SCHEMA_FILTER: {
+				Name:    OPT_SCHEMA_FILTER,
+				Desc:    "Excludes performance_schema, information_schema, mysql, and sys",
+				Default: "yes",
```

Comment:
Default "performance_schema, information_schema, mysql, and sys" (see `size.database`), so if user sets option they totally override default and set their own filter. E.g.if they want to do just +foo, then they'll have to do
```yaml
exclude: "performance_schema, information_schema, mysql, sys, foo"
```

But let's think about this more holistically because `size.table` is a little more complicated than `size.database`: ideally, we want to allow user to filter by schema _and/or_ table. E.g. what if they just want to exclude one specific table? 🤔 

Let's do similar to `size.database` but slightly different. So the options are:

```
	OPT_INCLUDE = "include"
	OPT_EXCLUDE = "exclude"
```

But the values can be:

* `foo` = any table named food (in any db)
* `foo.*` = all tables in db `foo`
* `foo.bar` = table `bar` in db `foo`

I think that makes sense in this context, and `foo.*` is  a MySQL thing (used for user grants). See what others on ODS think.

So really we'll have option `exclude` default to `performance_schema.*, information_schema.*, mysql.*, and sys.*`.